src/EditorFeatures/Core/AutomaticInsertionOfAbstractOrInterfaceMembers/AutomaticInsertionOfAbstractOrInterfaceMembersOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.AutomaticInsertionOfAbstractOrInterfaceMembers
+{
+    internal static class AutomaticInsertionOfAbstractOrInterfaceMembersOptions
+    {
+        // This value is only used by Visual Basic, and so is using the old serialization name that was used by VB.
+        public static readonly PerLanguageOption2<bool> AutomaticInsertionOfAbstractOrInterfaceMembers = new(
+            "visual_basic_automatic_insertion_of_abstract_or_interface_members_enabled", defaultValue: true);
```

Comment:
>visual_basic_automatic_insertion_of_abstract_or_interface_members_enabled [](http://example.com/codeflow?start=13&length=73)

Should the option be in VisualBasic project then?

Reply:
Let me check.. if it is only referenced in VB project then it might be OK

Reply:
Yup, it's possible, I will do the same for all the C# and VB options. (If they are only referenced in the shared project I will move them to its own language project)